pkg/proxy/apis/config/fuzzer/fuzzer.go

```diff
@@ -33,19 +33,22 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 	return []interface{}{
 		func(obj *kubeproxyconfig.KubeProxyConfiguration, c fuzz.Continue) {
 			c.FuzzNoCustom(obj)
-			obj.BindAddress = fmt.Sprintf("%d.%d.%d.%d", c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(256))
+			obj.NodeIPOverride = []string{fmt.Sprintf("%d.%d.%d.%d", c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(256))}
 			obj.ClientConnection.ContentType = c.RandString()
-			obj.Conntrack.MaxPerCore = ptr.To(c.Int31())
-			obj.Conntrack.Min = ptr.To(c.Int31())
-			obj.Conntrack.TCPCloseWaitTimeout = &metav1.Duration{Duration: time.Duration(c.Int63()) * time.Hour}
-			obj.Conntrack.TCPEstablishedTimeout = &metav1.Duration{Duration: time.Duration(c.Int63()) * time.Hour}
+			obj.Linux.OOMScoreAdj = ptr.To(c.Int31())
+			obj.Linux.Conntrack.MaxPerCore = ptr.To(c.Int31())
+			obj.Linux.Conntrack.Min = ptr.To(c.Int31())
+			obj.Linux.Conntrack.TCPCloseWaitTimeout = &metav1.Duration{Duration: time.Duration(c.Int63()) * time.Hour}
+			obj.Linux.Conntrack.TCPEstablishedTimeout = &metav1.Duration{Duration: time.Duration(c.Int63()) * time.Hour}
 			obj.FeatureGates = map[string]bool{c.RandString(): true}
-			obj.HealthzBindAddress = fmt.Sprintf("%d.%d.%d.%d:%d", c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(65536))
+			obj.HealthzBindAddresses = []string{fmt.Sprintf("%d.%d.%d.%d/32", c.Intn(256), c.Intn(256), c.Intn(256), c.Intn(256))}
```

Comment:
@danwinship should we run two instances of healthz/metrics server if the user provides a valid dual stack CIDR list and the node has local IPs for both the IP families?

Reply:
Two healthz servers would undo your previous step toward getting dual-stack health working. You should extend `ProxierHealthServer` to support multiple bind addresses the same way `ServiceHealthServer` does.

Likewise with the metrics server in `serveMetrics`.